__tests__/app.test.js

```diff
@@ -108,4 +107,57 @@ describe('api', () => {
               });
     });
   });
+
+  describe.only('GET /api/articles/:article_id/comments', () => {
+    test('status: 200, returns an object with key \'comments\' with an array of comment objects as the value', () => {
+      return request(app)
+              .get('/api/articles/3/comments')
+              .expect(200)
+              .expect('Content-Type', 'application/json; charset=utf-8')
+              .then(({ body }) => {
+                const comments = body.comments;
+                expect(comments.length).toBe(2);
+                comments.forEach((comment) => {
+                  expect(comment).toMatchObject({
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    article_id: 3,
+                    created_at: expect.any(String),        
+                  });
+                })
+              })
+    });
+
+    test('status: 200, comments are ordered by most recent', () => {
+      return request(app)
+              .get('/api/articles/1/comments')
+              .expect(200)
+              .expect('Content-Type', 'application/json; charset=utf-8')
+              .then(({ body }) => {
+                const comments = body.comments;
+                expect(comments.length).toBe(11);
+                expect(comments).toBeSortedBy('created_at', { descending : true })
+                })
+    });
+
```

Comment:
Please add a test for no comments for valid article ID, expecting 200 and an empty array.